packages/transformers/babel/src/env.js

```diff
@@ -48,7 +48,7 @@ export default async function getEnvOptions(
 
 function getNeededPlugins(targets: BabelTargets): Array<PresetEnvPlugin> {
   return presetEnv(
-    {assertVersion: () => true},
+    {version: '7.13.0', assertVersion: () => true}, // version is required since https://github.com/babel/babel/pull/12934/
```

Comment:
This should be 7.12.0 (https://github.com/parcel-bundler/parcel/blob/806cc65e0e430b1120cdcf1676e2f58738326d30/packages/transformers/babel/package.json#L43), or `require("@babel/core").version`.

Ideally, the safest approach would be `{ ...require("@babel/core"), assertVersion: () => true }`.